src/models/user.model.test.ts

```diff
@@ -0,0 +1,276 @@
+
+import { QueryIterator, ScanIterator } from '@aws/dynamodb-data-mapper'
+import mapper from '../db/dynamoDbClient'
+import User from './user.model'
+
+jest.mock('../db/dynamoDbClient')
+
+describe('User suit', () => {
+
+  beforeEach(() => {
+    //const mapperMock = jest.spyOn(mapper, 'put')
+  })
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+
+  describe('Create an user', () => {
+    it('Create user successfully', async () => {
+      //GIVEN
+      const userExpected = {
+        id: 12345,
+        name: 'name',
+        dob: '2007-02-03',
+        address: 'Tv 4 # 54-56',
+        description: 'test user',
+        imageUrl: 'www.testuser.com',
+        createdAt: '2021-05-10',
+        updateAt: '2021-05-10'
+      }
+      const user = new User('name',
+        new Date(2007, 2, 3),
+        'Tv 4 # 54-56',
+        'test user',
+        'www.testuser.com')
```

Comment:
Do you see this with better readibility than
```javascript
const user = {
  name: 'name',
  address: 'test user',
  imageUrl: 'www.testuser.com'
  dob: new Date(2007, 2, 3)
}
```

Reply:
Yes, is cleaner and easily to read.